src/Agentic.Workflow.Benchmarks/README.md

```
@@ -0,0 +1,274 @@
+### Key Metrics
+
+| Metric | Description | Target |
+|--------|-------------|--------|
+| **Mean** | Average execution time | Primary comparison metric |
+| **P95** | 95th percentile latency | Tail latency indicator |
+| **Gen0** | Gen0 garbage collections | Lower is better (0 ideal) |
+| **Allocated** | Memory allocated per operation | Lower is better (0 B ideal) |
+
+### Example Output
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)